src/components/ErrorModal.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../styles/ErrorModal.scss'
+
+const ErrorModal = props => {
+  return (
+    <section className="error-modal">
+      <section className="error-modal-box">
+        <h2 className="error-message">{props.message}</h2>
+      </section>
```

Comment:
Add a page reload button here

Reply:
What a nice way to display dynamic error messages!